src/util/crypto.ts

```diff
@@ -2,16 +2,9 @@
  * @module util/crypto
  */
 
-import {keccakAsU8a} from "@polkadot/util-crypto";
+import {bytes, bytes32,} from "../types";
+import {hash as hash256} from "@chainsafe/ssz/lib/util/hash";
```

Comment:
I think we should just export `hash` from ssz so we don't have to redeclare it below